FFaCiL/EllipticCurve.lean

```diff
@@ -143,6 +189,11 @@ def double (p : ProjectivePoint C) : ProjectivePoint C :=
 instance : Neg $ ProjectivePoint C where
   neg := fun ⟨x, y, z⟩ => ⟨x, -y, z⟩
 
+/--
+Addition in the Abelian group of projective points.
+The implementation is based on 
+Handbook of elliptic and hyperelliptic curve cryptography by Henri Cohen, et al., 13.2.1.b.
```

Comment:
Is this from the handbook or the https://eprint.iacr.org/2015/1060.pdf paper?

Reply:
I can't remember, TBH

Reply:
Oh, it seems you're right